docs/tags/common-schema.md

```diff
@@ -183,3 +183,78 @@ Below are tags that are reserved for future use:
 | service.type | Service type e.g. `hadoop` |
 | span.id | Building block of a trace in distributed tracing |
 | trace.id | Building block of a trace in distributed tracing |
+
+## Events Tags
+
+### Fields
+
+When you create an event you can send a ```JSON``` document which consists of multiple
```

Comment:
@fulyauluturk note how we use both the word "tags" and "fields".  Maybe we need to stick with one?